src/tracker_service.py

```diff
@@ -0,0 +1,254 @@
+        self,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> Vision.Properties:
+        return Vision.Properties(
+            classifications_supported=True,
+            detections_supported=True,
+            object_point_clouds_supported=False,
+        )
+
+    async def capture_all_from_camera(
+        self,
+        camera_name: str,
+        return_image: bool = False,
+        return_classifications: bool = False,
+        return_detections: bool = False,
+        return_object_point_clouds: bool = False,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ):
+        if not camera_name == self.camera_name:
+            raise ValueError(
+                "The camera_name %s doesn't match the camera_name configured for the tracker: %s."
+                % (camera_name, self.camera_name)
+            )
+        img = None
+        if return_image:
+            img = await self.camera.get_image(mime_type=CameraMimeType.JPEG)
+
+        classifications = None
+        if return_classifications:
+            if await self.tracker.is_new_object_detected():
+                classifications = [
+                    Classification(class_name="new_object_detected", confidence=1)
+                ]
+
+        detections = None
+        if return_detections:
+            detections = self.tracker.get_current_detections()
+
+        # do_cmd_res = None
+        # if extra is not None:
+        #     do_cmd_res = await self.do_command(extra)
+
+        return CaptureAllResult(
+            image=img, classifications=classifications, detections=detections
+        )
+
+    async def get_object_point_clouds(
+        self,
+        camera_name: str,
+        *,
+        extra: Optional[Dict[str, Any]] = None,
+        timeout: Optional[float] = None,
+        **kwargs,
+    ) -> List[PointCloudObject]:
+        raise NotImplementedError
+
+    async def get_detections(
+        self,
+        image: ViamImage,
+        *,
+        extra: Mapping[str, Any],
+        timeout: float,
+    ) -> List[Detection]:
+        return NotImplementedError
+
+    async def get_classifications(
+        self,
+        image: ViamImage,
+        count: int,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Classification]:
+        return NotImplementedError
+
+    async def get_classifications_from_camera(
+        self,
+        camera_name: str,
+        count: int,
+        *,
+        extra: Optional[Mapping[str, Any]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Classification]:
+        if not camera_name == self.camera_name:
+            raise ValueError(
+                "The camera_name doesn't match the camera_name configured for the tracker."
+            )
+        if await self.tracker.is_new_object_detected():
```

Comment:
Is this scoped? Can we delete this?

Reply:
you know it's not scoped. deleting!!!!!!